app/models/task_file.rb

```diff
@@ -6,6 +6,8 @@ class TaskFile < ApplicationRecord
   has_one_attached :attachment
   validates :name, presence: true
   validates :attachment, presence: true, if: -> { use_attached_file == 'true' }, on: :force_validations
+  validates :xml_id, presence: true
```

Comment:
If we force the XML ID to be present, shouldn't we always display it? And what about (a migration for) existing data? We could clear the test data, but probably should consider this as well.

Reply:
Indeed, existing exercises break:

<img width="634" alt="Bildschirm­foto 2022-12-17 um 10 38 36" src="https://user-images.githubusercontent.com/7300329/208235760-866be8f9-22c0-4e47-b857-69528d607bf9.png">

```
ActiveRecord::RecordInvalid - Validation failed: Files xml can't be blank, Tests files xml can't be blank, Model solutions files xml can't be blank
```